src/api-provider.ts

```diff
@@ -56,32 +58,32 @@ export class ApiProvider {
             return this.targetprocessApi;
         }
 
-        const config = this.getTargetprocessConfig();
+        const { username, password, subdomain } = this.getTargetprocessConfig();
 
-        if (config === null) {
+        if (username === null || password === null || subdomain === null) {
             log.error("Targetprocess authentication not configured correctly.");
             log.error("Use `twotime auth` to authenticate.");
             process.exit(1);
         }
 
-        const api = new Targetprocess("neworbit", config.username, config.password);
+        const api = new Targetprocess(subdomain, username, password);
 
         this.targetprocessApi = api;
 
         return this.targetprocessApi;
     }
 
-    public setHarvestConfig(accessToken: string, accountId: number) {
+    public setHarvestConfig(accessToken: string, accountId: number, ) {
         const config: HarvestConfig = {
             accessToken, accountId
         };
 
         this.store.set(CONFIG_KEYS.HARVEST, config);
     }
 
-    public setTargetprocessConfig(username: string, password: string) {
+    public setTargetprocessConfig(username: string, password: string, subdomain: string) {
```

Comment:
I feel like we shouldn't have 3 string arguments, I'd change this to an interface or something to encapsulate the three arguments